src/containers/shared/components/Amount.tsx

```diff
@@ -26,16 +26,16 @@ export const Amount = ({
 
   return (
     <span className="amount">
-      <span className="amount-localized">
-        {modifier && <span className="amount-modifier">{modifier}</span>}
-        {localizedAmount}
-      </span>{' '}
       <Currency
         issuer={displayIssuer ? issuer : ''}
         currency={currency}
         link
         displaySymbol={false}
-      />
+      />{' '}
```

Comment:
These changes should be backed out - the `Amount` file should not need to be modified

Reply:
Then token code will be on the right side. is it okay?

Reply:
Yes. This change will modify a lot of other pages, and we don't want that.